2025/webextensions-wg.html

```diff
@@ -66,378 +79,760 @@
+            specification.
+          </li>
+          <li>
+            <strong>Web Extensions APIs</strong> - the interfaces that
+            extensions are built on.
+          </li>
+          <li>
+            <strong>Resource files and formats</strong> - assets created by
+            third-party developers that are consumed by the user agent. Assets
+            include, but are not limited to, a manifest format and files
+            specifying how a given extension is localized. The extension
+            manifest format predates and is notably different from the
+            <a href="https://www.w3.org/TR/appmanifest/"
+              >Web Application Manifest</a
+            >
+            specification.
+          </li>
+          <li>
+            <strong>Native Messaging</strong> - API surface extensions to
+            exchange messages with native applications running on the host
+            system.
+          </li>
+          <li>
+            <strong>WebDriver integration</strong> - the production of artifacts
+            necessary to enable implementors to verify their implementations
+            conform to a common test suite, and to enable extension developers
+            to create automated tests to verify extension behaviors across user
+            agents.
+          </li>
+          <li>
+            <strong>Extension/Web interactions</strong> - concepts, systems, and
+            interfaces that allow extensions to interact with web pages, or that
+            restrict extensions from interacting with web pages. Those include
+            both direct interactions (such as modifying an open page's DOM) and
+            indirect interactions (such as using a WebExtensions API to clear
+            browser history).
+          </li>
+          <li>
+            <strong>Network request modification</strong> - as an example of a
+            deep integration point: user agents may allow extensions to block,
+            allow, or mutate network requests initiated by other parts of the
+            user agent, such as an open web page. This capability has
+            implications on aspects of the web platform including, but not
+            limited to, authentication, identity, navigation, and session
+            management.
+          </li>
         </ul>
 
         <section id="section-out-of-scope">
           <h3 id="out-of-scope">Out of Scope</h3>
-            <p>The following features are out of scope, and will not be addressed by this Working group.</p>
+          <p>
+            The following features are out of scope, and will not be addressed
+            by this Working group.
+          </p>
 
-            <ul class="out-of-scope">
-              <li>Deployment and distribution mechanisms, including but not limited to:
+          <ul class="out-of-scope">
+            <li>
+              Deployment and distribution mechanisms, including but not limited
```

Comment:
We should revise this to allow for packaging discussion as it relates to conformance testing.

Reply:
Resolved in https://github.com/w3c/charter-drafts/pull/711/commits/22ef9825196862e83ac642cf60803baec894389f.